src/app/tests/suites/certification/Test_TC_BR_1.yaml

```diff
@@ -326,9 +315,8 @@ tests:
 
           ./chip-tool  descriptor read device-type-list 1 4
 
+          Verify the DeviceTypeList entries with endpoint 1  On TH(chip-tool) Log
```

Comment:
It should be 

Verify the DeviceTypeList entries with endpoint 4 On TH(chip-tool) Log